tools/optimize_svg.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+import argparse
+import tempfile
+import shutil
+from scour import scour
+
+ARGS = (
+    "--enable-comment-stripping",
+    "--enable-id-stripping",
+    "--enable-viewboxing",
+    "--indent=none",
+    "--no-line-breaks",
+    "--remove-descriptions",
+    "--remove-metadata",
+    "--remove-titles",
+)
+
+
+def main(argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input_file", nargs="+")
+    args = parser.parse_args(argv)
+    options = scour.parse_args(list(ARGS))
+
+    for input_file in args.input_file:
+        with open(input_file, mode="rb") as input:
+            with tempfile.NamedTemporaryFile(
+                suffix=".svg", mode="wb", delete=False
+            ) as output:
+                scour.start(options, input, output)
+        shutil.move(output.name, input_file)
```

Comment:
Can we process them in parallel?

Reply:
Probably not really worth the trouble.

Reply:
I mean it takes a little while if you with `--all-files`, but usually only the files you commit are checked.

But I can look into it.

Reply:
Thats probably already too much work. I guess it would also possibly introduce bugs if some entries are separate for some reason, etc. Disregard the first comment.

Reply:
Too late, check the latest commit :D